### PR TITLE
refactor: return filename as part of viewer info

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds the field `file_name` to the submission-related info to be returned when getting the Turnitin viewer URL.

### CODE REVIEW

File: platform_plugin_turnitin/api/v1/views.py
Comment (BryanttV) [inline platform_plugin_turnitin/api/v1/views.py:1]: Please, update the docstring where appropriate.

**Review by BryanttV — APPROVED** [review]

File: platform_plugin_turnitin/api/v1/views.py
Comment (mariajgrimaldi) [reply platform_plugin_turnitin/api/v1/views.py:1]: Done. Thanks!

Comment (BryanttV) [inline platform_plugin_turnitin/api/v1/views.py:1]: @mariajgrimaldi, could you run `make format`?

Comment (mariajgrimaldi) [reply platform_plugin_turnitin/api/v1/views.py:1]: Done! Thank you

**Pull request merged**